theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -15,9 +15,16 @@ let fundingAgenciesApp = new Vue({
     crossrefFunders: [], // array of funders to be filled from crossref api
     crossrefFundersNames: [],
     crossrefSelected: false,
-    CROSSREF_API_URL: "https://api.crossref.org/funders?query=:query",
+    HELP_EMAIL: "help@cuahsi.org",
+    CROSSREF_API_URL: "https://api.crossref.org/funders",
+    LIMIT_US: true, // limit to US funders
     MIN_SEARCH_LEN: 3, // min # of chars before running a query
-    DEBOUNCE_API_MS: 500, // debounce api requests
+    // bug in vue-bootstrap-typeahead, this parameter doesn't seem to have any impact
+    // TODO: vue-bootstrap-typeahead is deprecated.
+    // migrate to vue-boostrap-autocomplete or similar
+    // https://github.com/drikusroor/vue-bootstrap-autocomplete
+    MAX_MATCHES: 50, // max number of matches to show
```

Comment:
this PR resolves querying the API but the vue-bootstrap-typeahead does not display the full set

Reply:
I'm not aware of any way to use esm modules like https://github.com/drikusroor/vue-bootstrap-autocomplete for Vue2 non-build static vue apps like the [funding-agency-app](https://github.com/hydroshare/hydroshare/blob/develop/theme/static/js/hs-vue/funding-agency-app.js). @Maurier that's not possible right?

Reply:
I'm afraid this is not possible. They would have to provide the files (script and CSS) via a CDN.

You could try your luck by cloning their repository and building the project and see which files it generates. If they export a UMD build you might be able to use it as in their demo at https://github.com/alexurquhart/vue-bootstrap-typeahead/blob/master/dist/demo.html.

Reply:
thanks! their bundler does export a UMD but when I import it into the HS template it doesn't seem to register any of the components.

Okay thanks Mauriel, looks like we will have to stick with what we have for now